app/org/sagebionetworks/bridge/services/email/ParticipantRosterProvider.java

```diff
@@ -81,6 +81,9 @@ String createParticipantTSV() {
         for (String attribute : study.getUserProfileAttributes()) {
             append(sb, StringUtils.capitalize(attribute), true);
         }
+        if (study.isHealthCodeExportEnabled()) {
+            append(sb, "Health Code", true);
```

Comment:
Only place this is used in a CSV export file. Seems cleaner to use the string inline rather than obfuscate things by pulling it out to a static string at the top.